FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Health.Fhir.Synapse.HealthCheck.Checkers
 {
     public class AzureContainerRegistryHealthChecker : BaseHealthChecker
     {
-        private const string MediatypeV2Manifest = "application/vnd.docker.distribution.manifest.v2+json";
+        private const string ManifestMediatype = "application/vnd.oci.image.manifest.v1+json,application/vnd.docker.distribution.manifest.v2+json";
```

Comment:
Add two variables here? and add reference to converter?

Reply:
This is for hotfix. We can refine this later